pipeline_tools/tests/shared/submission/test_create_file_descriptor.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pytest
+
+import pipeline_tools.shared.submission.create_file_descriptor as cfd
+from pathlib import Path
+
+
+@pytest.fixture(scope='module')
+def test_data():
+    class Data:
+        file_path = '/fake/file/path.fasta'
+        size = 1000
+        sha256 = '12998c017066eb0d2a70b94e6ed3192985855ce390f321bbdb832022888bd251'
+        crc32c = '0b83b575'
+        raw_schema_url = 'http://schema.humancellatlas.org/'
+        file_descriptor_schema_version = '2.0.0'
+        creation_time = '2020-08-10T14:24:26.174274-07:00'
+
+    return Data
+
+
+@pytest.fixture
+def data_file():
+    def _data_file(file_name):
+        return (
+            f'{Path(os.path.split(__file__)[0]).absolute().parents[1]}/data/{file_name}'
+        )
+
+    return _data_file
+
+
+class TestCreateAnalysisMetadata(object):
```

Comment:
Did you mean `TestCreateFileDescriptor` here?

Reply:
Absolutely. Good catch!